package/src/layouts/Layout.astro

```diff
@@ -6,6 +6,10 @@ import { getPublication } from '../hn-gql'
 import { SEO } from "astro-seo";
 import { AstroFont } from "astro-font";
 import type { AstroHashnodeLayoutProps } from '../proptypes/layouttypes'
+import { ViewTransitions } from 'astro:transitions';
+import config from "virtual:astro-hashnode/config";
+
+const useTranstions:boolean = !config.disableViewTransitions;
```

Comment:
Yeah so if this option was positively worded you wouldn't need this `!` here

Reply:
The Example above shows the default option.  Im inverting it to use down below since the default is false.  as DO NOT disable ViewTransitions